tasks.py

```diff
@@ -341,6 +341,20 @@ def post_upgrade(context):
     run_command(context, command)
 
 
+@task()
+def dumpdata(context, format="json"):
+    """Dump data from database to db_output file."""
+    command = f"nautobot-server dumpdata -e extras.job --indent 4 -o db_output.{format} --format {format}"
+    run_command(context, command)
```

Comment:
In the case where `local` is False (Docker-based dev environment), these commands presumably need to copy the file out of (and back into) the Docker container to/from the local filesystem.

Reply:
As long as the file is in the root of the project it will be passed over to the local system.

Reply:
This works on Mac, IIRC, but on a Linux system I believe the file will be owned by the `nautobot` user account inside Docker, which may not correspond to an account on the host system. I think it's better to create it inside the container and do a `docker cp` to the local filesystem?

Reply:
@carbonarok